tests/ui_tests/app/page/admin/page.py

```diff
@@ -489,3 +510,83 @@ def click_delete_button(self):
+            self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.role_select)
+            self.wait_element_visible(AdminPoliciesLocators.AddPolicyPopup.FirstStep.role_item)
+            for available_role in self.find_elements(AdminPoliciesLocators.AddPolicyPopup.FirstStep.role_item):
+                if available_role.text == role:
+                    available_role.click()
+                    break
+        assert users or groups, "There are should be users or groups in the policy"
+        if users:
+            with allure.step(f"Select users {users} in popup"):
+                self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.users_select)
+                self.wait_element_visible(AdminPoliciesLocators.AddPolicyPopup.FirstStep.users_item)
+                available_users = self.find_elements(AdminPoliciesLocators.AddPolicyPopup.FirstStep.users_item)
+                for user in users.split(","):
+                    for available_user in available_users:
+                        if available_user.text == user:
+                            self.scroll_to(available_user)
+                            available_user.click()
+                self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.users_select)
+        if groups:
+            with allure.step(f"Select groups {groups} in popup"):
+                self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.group_select)
+                self.wait_element_visible(AdminPoliciesLocators.AddPolicyPopup.FirstStep.group_item)
+                available_groups = self.find_elements(AdminPoliciesLocators.AddPolicyPopup.FirstStep.group_item)
+                for group in groups.split(","):
+                    for available_group in available_groups:
+                        if available_group.text == group:
+                            self.scroll_to(available_group)
+                            available_group.click()
+                self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.group_select)
+        self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.FirstStep.next_btn_first)
+        self.wait_element_visible(AdminPoliciesLocators.AddPolicyPopup.SecondStep.next_btn_second)
+        self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.SecondStep.next_btn_second)
+        self.wait_element_visible(AdminPoliciesLocators.AddPolicyPopup.ThirdStep.create_btn)
+        self.find_and_click(AdminPoliciesLocators.AddPolicyPopup.ThirdStep.create_btn)
+        self.wait_element_hide(AdminPoliciesLocators.AddPolicyPopup.block)
+
+    def get_all_policies(self) -> [AdminGroupInfo]:
+        """Get all policies info."""
+
+        policies_items = []
+        policies_rows = self.table.get_all_rows()
+        for policy in policies_rows:
+            policies_items.append(policy.text)
+        return policies_items
+
+    def select_all_groups(self):
+        self.find_elements(self.table.locators.header)[0].click()
```

Comment:
Did I get it right: if you click the first table header, it will select all groups? Can you please mention it in the docstring?

Reply:
And I guess it's selecting all policies, not groups?